pkgs/games/rigsofrods/default.nix

```diff
@@ -4,14 +4,14 @@
   }:
 
 stdenv.mkDerivation rec {
-  version = "git-20160412";
+  version = "0.4.7.0";
   name = "rigsofrods-${version}";
 
   src = fetchFromGitHub {
     owner = "RigsOfRods";
     repo = "rigs-of-rods";
-    rev = "1ebd359dbd467b4c3171dd6d054e7d8ec39f78ba";
-    sha256 = "0h71nrgq5r5cnh20c7wl8jzyaf50dj1b5jdrwihnklpsfyfvjlw4";
+    rev = "81c2f9392933bbef0ad007c25489d4db7a1cf27d";
```

Comment:
I seem to remember some talk of fetchFromGitHub's output varying if a tag name is used instead of a commit hash, that's why I did it this way. I'm not sure though, I might be remembering wrong — I'll change it.